src/test/components/TimelineMarkers.test.js

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import React from 'react';
+import TimelineMarkers from '../../content/containers/TimelineMarkers';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import mockCanvasContext from '../fixtures/mocks/canvas-context';
+import { storeWithProfile } from '../fixtures/stores';
+import { getProfileWithMarkers } from '../store/fixtures/profiles';
+
+jest.useFakeTimers();
```

Comment:
> The timers are only fake for the scope of that file, 

Oh OK. Jest is different to mocha in this regard. Good!

about my comment about the fake timers, I missed the call to `jest.runAllTimers();`, so all is good :)